test/exitHandler.js

```diff
@@ -303,6 +330,59 @@ contract('ExitHandler', (accounts) => {
         
       });
 
+      it('Should allow to challenge limbo exit and prevent it', async () => {
+        // utxo that will have spend exit utxo
+        const period = await submitNewPeriod(depositTx);
+        const spendTx = Tx.transfer(
+          [new Input(new Outpoint(transferTx.hash(), 0))],
+          [new Output(50, charlie)]
+        ).sign([bobPriv]);
+
+        const depositProof = period.proof(depositTx);
+        const outputIndex = 1;
+        const exitStake = exitHandler.exitStake();
+        const signature = (transferTx + c).sign(alicePriv);
+        
+        const txs = [spendTx];
+        const block = txs.reduce(
+          (b, tx) => b.addTx(tx),
+          new Block(33)
+        );
+        prevRoot = period.merkleRoot();
```

Comment:
I need spendProof (Proof that bob sent the money to charlie) so that i can pass it to `challengeLimbo` and  validate it there. Can i create a block simply and not publish it to `Bridge`  so that i can get period.